### PR TITLE
Ddcnl 5628

### PR DESCRIPTION
**Pull request by Sachin300793**

### CODE REVIEW

File: app/views/tags/cardNewsAndUpdates.scala.html [hunk 1]
```diff
@@ -30,6 +30,8 @@
     </@headingTag>
     <ul>
         <li><a class="ga-track-anchor-click"  href="@controllers.routes.InterstitialController.displayNewsAndUpdates()#verify" data-ga-event-category="link - click" data-ga-event-action="Income" data-ga-event-label="HMRC Personal Tax Account to stop using GOV.UK Verify">@messages("label.stop_using_Verify")</a></li>
+        <br>
```
Comment (pascal22p) [inline app/views/tags/cardNewsAndUpdates.scala.html:33]: That `<br>` tag should not be here I think.

**Review by Luka-Abey — APPROVED** [review]

**Review by pascal22p — CHANGES_REQUESTED** [review]

File: app/views/tags/cardNewsAndUpdates.scala.html (hunk 1 above)
Comment (Muhammad-Ali-Ahmad) [reply app/views/tags/cardNewsAndUpdates.scala.html:33]: I agree

Comment (Sachin300793) [reply app/views/tags/cardNewsAndUpdates.scala.html]: Fixed it now , thank you both

**Review by pascal22p — APPROVED** [review]

**Review by Christian-QA — APPROVED** [review]

File: conf/messages [hunk 2]
```diff
@@ -483,6 +483,7 @@ label.class_1_paid_by_employees=Class 1 (paid by employees)
 label.class_4_paid_by_self_employed=Class 4 (paid by self-employed)
 label.secondary_class_1_1a_1b_paid_by_employers=secondary Class 1, 1A and 1B (paid by employers)
 label.prepare_for_the_health_and_social_care_levy=Prepare for the Health and Social Care Levy (opens in new tab)
+healthAndSocialCareLevy.link.url = https://www.gov.uk/guidance/prepare-for-the-health-and-social-care-levy
```
Comment (Christian-QA) [inline conf/messages:486]: Shouldn't this be in the ConfigDecorator?

File: conf/messages.cy [hunk 3]
```diff
@@ -303,6 +303,7 @@ label.class_1_paid_by_employees=CYG Dosbarth 1 (a delir gan gyflogeion)
 label.class_4_paid_by_self_employed=CYG Dosbarth 4 (a delir gan bobl hunangyflogedig)
 label.secondary_class_1_1a_1b_paid_by_employers=CYG Dosbarth 1, 1A ac 1B eilaidd (a delir gan gyflogwyr)
 label.prepare_for_the_health_and_social_care_levy=Paratoi ar gyfer yr Ardoll Iechyd a Gofal Cymdeithasol (yn agor tab newydd)
+healthAndSocialCareLevy.link.url = https://www.gov.uk/guidance/prepare-for-the-health-and-social-care-levy.cy
```
Comment (Christian-QA) [inline conf/messages.cy:306]: Same as above but with `.concat("cy")` maybe?

File: conf/messages (hunk 2 above)
Comment (Sachin300793) [reply conf/messages:486]: the url for english and welsh content are different , so easiest way to put them in messages

File: conf/messages.cy (hunk 3 above)
Comment (Sachin300793) [reply conf/messages.cy:306]: the url for english and welsh content are different , so easiest way to put them in messages

**Review by Christian-QA — APPROVED** [review]

**Review by pascal22p — APPROVED** [review]

**Review by pascal22p — APPROVED** [review]

**Pull request merged**